src/components/BankTransactionRow/BankTransactionRow.tsx

```diff
@@ -323,17 +328,17 @@ export const BankTransactionRow = ({
                       setTransactionCategory(bankTransaction.id, selectedCategory)
                       setShowRetry(false)
                     }}
-                    isLoading={bankTransaction.processing}
+                    isDisabled={bankTransaction.processing}
```

Comment:
We don't check if categorization is enabled here like we do in the `ExpandedBankTransactionRow`. I _think_ that's because we don't need to this combobox already isn't displayed if categorization is enabled, but double checking in case.